CHANGELOG

```diff
@@ -1,6 +1,10 @@
 * Fri Feb 09 2018 Liz Nemsick <lnemsick.simp@gmail.com> - 7.4.2-0
 - Add the missing puppetlabs/inifile dependency to the metadata.json
 - Fixed puppet lint problem
+- pupmod::master::simp_auth
```

Comment:
Should be in its own changelog entry, since significant work done by a different author.